### ISSUE TITLE
auto-update "Updated" dates from in-list values

### ISSUE THREAD

**Issue by DandelionSprout** [collinbarrett/FilterLists#439]
Many filterlists have info tags inside them á la `! Last updated:`, `! Last modified:`, `! Updated:`, `# Updated`, `# Generated on`, `# TimeUpdated:`, `! TimeUpdated:`, and so on. And those dates are almost always pretty accurate indicators for when a list was last updated. Therefore I propose that the `Updated` values on FilterLists.com could be auto-updated with such values whenever they are available.

There are admittedly a lot of date formats that'd need to be accounted for, e.g. `05 Sep 2018 (…)`, `05September2018(…)`, `September 5, 2018`, `2018-09-05 (…)`, and probably some other ones as well.

Edit: I also propose that lists whose such values are more than 730 days (2 non-leap years) ago could've had those dates auto-added to the "Discontinued" values as well, but that'd be merely a secondary bonus.

**Comment by collinbarrett** [comment]
closing as duplicate of #202 

Capturing these dates is certainly on the roadmap. I am unsure whether we will use them as the primary source of the last updated date, however. I have found that some lists auto-update their date on a schedule even if the list hasn't actually changed at all.

However, in the latest changes to SnapshotService, I am determining if a list has been updated by its md5 hash, which will be different even if only the date metadata has changed. So, I guess that isn't as relevant anymore.
